contracts/governance/GovernanceStaking.sol

```diff
@@ -123,7 +122,7 @@ contract GovernanceStaking is
 	 * @dev Returns the number of decimals used to get its user representation.
 	 */
 	function decimals() public view virtual override returns (uint8) {
-		return 2;
+		return 18;
```

Comment:
this doesnt change anything, because it is not used anywhere.
the user stakes G$ which are 2 decimals, the amount of LP tokens the user get for staking his G$ is exactly equal to the G$ amount he staked. so it makes no sense to set the LP token to 18 decimals.
The amount of G$ staked is the value which is passed to increase/decrease productivity, if you would have wanted to increase precision you would have needed to _convertTo18Decimals the amount sent to increase/decrease just like you did in stakers distribution.
but when withdrawing you would need to scale it back down to the 2 decimals of G$